docs/en/solutions/Leveraging_Knative_Eventing_for_Cloud_Event_Publishing_and_Consuming.md

```
@@ -0,0 +1,813 @@
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: high-value-orders
+  namespace: my-app-namespace
+spec:
+  broker: default
+  filters:
+  - all:
+	- exact:
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)